.trivyignore

```diff
@@ -1,18 +1,8 @@
 # CRITICAL
-CVE-2022-48174
```

Comment:
Hey @rashmichandrashekar Can you update the release notes with this PR too? otherwise looks good to me.

Reply:
thanks for catching, let me do that.